app/src/main/java/com/lagradost/cloudstream3/ui/player/GeneratorPlayer.kt

```diff
@@ -106,6 +113,8 @@ class GeneratorPlayer : FullScreenPlayer() {
 
     override fun embeddedSubtitlesFetched(subtitles: List<SubtitleData>) {
```

Comment:
Might be more valid to have 2 different callbacks or one callback called onEmbbededDataFetched or smth

Reply:
other than that, everything looks fine. if you feel like this is not an issue this can be merged now